website/src/components/DataUseTerms/EditDataUseTermsModal.tsx

```diff
@@ -0,0 +1,287 @@
+        });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [sequenceFilter]);
+
+    const [state, setState] = useState<DataState>({ type: 'loading' });
+
+    useEffect(() => {
+        if (detailsHook.isLoading) {
+            return;
+        }
+        if (detailsHook.error !== null && state.type !== 'error') {
+            setState({ type: 'error', error: detailsHook.error });
+            return;
+        }
+        if (detailsHook.data) {
+            const newState = getLoadedState(detailsHook.data.data);
+            setState(newState);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [detailsHook.data, detailsHook.error, detailsHook.isLoading]);
+
+    return (
+        <>
+            <button className='mr-4 underline text-primary-700 hover:text-primary-500' onClick={openDialog}>
+                Edit data use terms
+            </button>
+            <BaseDialog title='Edit data use terms' isOpen={isOpen} onClose={closeDialog}>
+                {state.type === 'loading' && 'loading'}
+                {state.type === 'error' && `error: ${state.error}`}
+                {state.type === 'loaded' &&
+                    (accessToken === undefined ? (
+                        <p>You need to be logged in to edit data use terms.</p>
+                    ) : (
+                        <EditControl
+                            clientConfig={clientConfig}
+                            accessToken={accessToken}
+                            state={state}
+                            closeDialog={closeDialog}
+                            sequenceFilter={sequenceFilter}
+                        />
+                    ))}
+            </BaseDialog>
+        </>
+    );
+};
+
+interface EditControlProps {
+    clientConfig: ClientConfig;
+    accessToken: string;
+    state: LoadedState;
+    sequenceFilter: SequenceFilter;
+    closeDialog: () => void;
+}
+
+const EditControl: React.FC<EditControlProps> = ({ clientConfig, accessToken, state, closeDialog, sequenceFilter }) => {
+    const [dataUseTerms, setDataUseTerms] = useState<DataUseTerms>({ type: openDataUseTermsType });
+
+    switch (state.resultType) {
+        case 'allOpen':
+            return (
+                <>
+                    <ActiveFilters sequenceFilter={sequenceFilter} />
+                    <p>All selected sequences are already open, nothing to edit.</p>
+                </>
+            );
+        case 'mixed':
+            return (
+                <div className='space-y-4'>
+                    <ActiveFilters sequenceFilter={sequenceFilter} />
+                    <p>
+                        {state.openCount} open and {state.restrictedCount} restricted sequences selected.
+                    </p>
+                    <p>
+                        You can release all the {state.restrictedCount} restricted sequences as open. If you want to
```

Comment:
```suggestion
                        You can release all the {state.restrictedCount} restricted sequences, moving them to the Open Data Use Terms. If you want to
```

Reply:
We should also make these descriptions of terms links that open in a new window to the terms (which are available in the Values.yaml, unsure if currently passed to the website)

Reply:
https://github.com/loculus-project/loculus/pull/3322/commits/4f25f4423a2a2cec8a7b512e7cf2693e874f5974

I used a link to the same place that the radio buttons link to. Neither that link nor the radio button links open in a new tab (I agree that it would probably be useful, but then for all of them, right?)

Reply:
Yes, they should probably both open in new tabs - sorry for slow reply